src/ComplexTypes/GetSignatureRequest.php

```diff
@@ -14,23 +14,23 @@ class GetSignatureRequest extends BaseType
     protected $Customer = null;
 
     /**
-     * @var RequestSignature $Shipment
+     * @var RequestShipment $Shipment
```

Comment:
Why exactly was this file edited? The signature request is part of the (already implemented) ShippingStatus client. I'm not aware of a bug with this functionality, but even if there is, it shouldn't be a part of this PR.